packages/fx-core/src/plugins/resource/appstudio/index.ts

```diff
@@ -278,6 +278,25 @@ export class AppStudioPlugin implements Plugin {
     return ok(localTeamsAppId);
   }
 
+  public async checkPermission(ctx: PluginContext): Promise<Result<any, FxError>> {
+    TelemetryUtils.init(ctx);
+    TelemetryUtils.sendStartEvent(TelemetryEventName.checkPermission);
+
+    try {
+      const checkPermissionResult = await this.appStudioPluginImpl.checkPermission(ctx);
+      TelemetryUtils.sendSuccessEvent(TelemetryEventName.checkPermission);
```

Comment:
Do we need to add some properties for the telemetry?

Reply:
As commented in aad plugin, telemetry will be added after we have conclusion with yanliang in another pr. Here I only add least properties.